gui/global.c

```diff
@@ -109,7 +109,9 @@ static int op_show_log_messages(int op)
   struct PagerData pdata = { 0 };
   struct PagerView pview = { &pdata };
 
-  pdata.fname = tempfile;
+  pdata.fname = buf_strdup(tempfile);
+
+  buf_pool_release(&tempfile);
```

Comment:
`buf_string()`, then `buf_pool_release()` after `mutt_do_pager()`

Reply:
should be fixed now